### PR TITLE
WIP: Fix latexify support

### PR DESCRIPTION
**Pull request by korsbo**
Latexification of MKT objects is broken. Here, I try to fix that. 

- [x] fix latexify for simple `Equation`s
- [ ] fix latexify for `Equation`s with higher-order derivatives #621
- [x] fix latexify for `System`s (currently poorly implemented and surrounded by `$$`).

### CODE REVIEW

**Comment by korsbo** [comment]
It would be good if someone who knows MTK could confirm that my changes to `toexpr` won't break anything. I have made minor corrections to the tests and I can't see any issues from there.

**Comment by korsbo** [comment]
https://github.com/korsbo/Latexify.jl/pull/120 fixes the latexificaition of Systems.

File: src/direct.jl [hunk 1]
```diff
@@ -225,7 +225,7 @@ function toexpr(O::Term)
           return Expr(:call, :^, Expr(:call, :inv, toexpr(O.args[1])), -(O.args[2].value))
       end
   end
-  return Expr(:call, O.op, toexpr.(O.args)...)
+  return Expr(:call, toexpr(O.op), toexpr.(O.args)...)
```
Comment (shashi) [inline src/direct.jl:228]: We can't do this because it is wrong namespacing

Comment (shashi) [reply src/direct.jl:228]: Try using `Base.isoperator(nameof(f))` to check if `f` is an infix

File: test/simplify.jl [hunk 2]
```diff
@@ -19,9 +19,9 @@ simplify(minus_op)
 
 @variables x
 
-@test toexpr(expand_derivatives(Differential(x)((x-2)^2))) == :($(*)(2, $(+)(-2, x)))
-@test toexpr(expand_derivatives(Differential(x)((x-2)^3))) == :($(*)(3, $(^)($(+)(-2, x), 2)))
-@test toexpr(simplify(x+2+3)) == :($(+)(5, x))
+@test toexpr(expand_derivatives(Differential(x)((x-2)^2))) == :(2 * (-2 + x))
+@test toexpr(expand_derivatives(Differential(x)((x-2)^3))) == :(3 * (-2 + x) ^ 2)
+@test toexpr(simplify(x+2+3)) == :(5 + x)
```
Comment (shashi) [inline test/simplify.jl:24]: Yeah this stuff was intentional

**Review by shashi — CHANGES_REQUESTED** [review]

**Comment by shashi** [comment]
Closing this in favor of #629 maybe a fix for #621 should come in a separate PR.

**Pull request closed without merging**